recipes/rb-net-http-persistent/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "net-http-persistent" %}
+{% set version = "4.0.1" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 2752f4cce05fd1c45e0537c6f3a98fa5a4899efd5f88e63c104ed5f05cbddef9
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
```

Comment:
Nit pick: We don't need this line since we don't extract the license from the Gem.